Project/Source/Components/ComponentTransform.cpp

```diff
@@ -125,7 +113,7 @@ void ComponentTransform::CalculateGlobalMatrix(bool force) {
 void ComponentTransform::SetPosition(float3 position_) {
 	position = position_;
 	InvalidateHierarchy();
-	for (Component* component : GetOwner().components) {
+	for (Component* component : GetOwner().GetComponents()) {
 		component->OnTransformUpdate();
```

Comment:
He hecho cambios de esto en mi pullRequest. Solo un reminder para que cambiemos el .GetComponents() cuando se haga el merge tambien.

(Lo que hice es poner este for en una funcion separada, que se llama recursivamente para los hijos del GameObject)

Reply:
Por eso te he pasado el review a ti, porque se que habías tocado esto. En principio tu haces el merge primero y luego yo. Lo tomo en cuenta.